### PR TITLE
chore: [#184305938] view more / less button added to formation review…

### PR DESCRIPTION
**Pull request by FarazA22**
## Description

I created a wrapper component that takes a string and can restrict the number of lines can wrap. The component has a view more and view less button that expands and collapses the text. 

### Ticket

[184305938](https://www.pivotaltracker.com/story/show/184305938)

### Approach

I leverage a popular yarn package.

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation, if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code

### CODE REVIEW

File: web/src/components/ExpandCollapseString.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  viewLessText: string;
+}
+
+export const ExpandCollapseString = (props: Props): ReactElement => {
+  return (
+    <div {...(props.dataTestId ? { "data-testid": props.dataTestId } : {})}>
+      <ClampLines
+        text={props.text}
+        id={props.id}
+        lines={2}
```
Comment (aloverso) [inline web/src/components/ExpandCollapseString.tsx:18]: In order for this to be more reusable, should the number of lines be a prop maybe?

File: web/src/components/ExpandCollapseString.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { ReactElement } from "react";
+import ClampLines from "react-clamp-lines";
+
+interface Props {
+  text: string;
+  id: string;
```
Comment (aloverso) [inline web/src/components/ExpandCollapseString.tsx:6]: Why is it necessary to pass an ID? we don't often use ids on html elements

File: web/src/components/tasks/business-formation/review/ReviewText.tsx [hunk 3]
```diff
@@ -14,6 +15,7 @@ interface Props {
   header: string;
   fieldName: FormationTextField;
   stepName: FormationStepNames;
+  isExpandCollapseStringEnabled?: boolean;
```
Comment (aloverso) [inline web/src/components/tasks/business-formation/review/ReviewText.tsx:18]: could maybe be shortened to `isExpandable` or something?

**Comment by aloverso** [comment]
I'm surprised this isn't failing the dependency-check, since it introduces a new library without updating the dependency checker file. Any idea why?

File: web/src/components/ExpandCollapseString.tsx (hunk 1 above)
Comment (FarazA22) [reply web/src/components/ExpandCollapseString.tsx:18]: I was debating whether they should be a prop now or in the future when the need arose. I can make the update now.

Comment (aloverso) [reply web/src/components/ExpandCollapseString.tsx:18]: Yeah, I'm usually opposed to extra props until the need arises, totally. I just thought maybe in this case, it was important enough to the usage of the component that it seemed like a good candidate

**Comment by FarazA22** [comment]
> I'm surprised this isn't failing the dependency-check, since it introduces a new library without updating the dependency checker file. Any idea why?

This is a really good question, I usually don't worry about the dependency check until it screams at me. I'll take a look.

**Pull request merged**